external/storm-kafka/src/jvm/org/apache/storm/kafka/PartitionManager.java

```diff
@@ -46,6 +46,9 @@
     private final CountMetric _fetchAPIMessageCount;
```

Comment:
Agree that constructor is somewhat bloated. The whole class can use some refactoring. can we do it in a separate PR?